README.rst

```diff
@@ -1,118 +1,87 @@
-flagged as a Template Repository, meaning it can be used as a basis for new
-GitHub repositories by clicking the green "Use this template" button above.
-The rest of this document describes how to work with your new micro-frontend
-**after you've created a new repository from the template.**
+This is the Catalog micro-frontend application, currently under development by `2U <https://2u.com>`_.
+
+**What is the domain of this MFE?**
+
+- Home / Index page
+- Course About page
+- Course Catalog page
+
+These are public-facing pages intended for unauthenticated users.
+The goal is to replace legacy views in ``edx-platform`` with modern, React and Paragon-based implementations.
 
 Getting Started
 ***************
 
-After copying the template repository, you'll want to do a find-and-replace to
-replace all instances of ``frontend-template-application`` with the name of
-your new repository.  Also edit index.html to replace "Application Template"
-with a friendly name for this application that users will see in their browser
-tab.
-
 Prerequisites
 =============
 
-The `devstack`_ is currently recommended as a development environment for your
-new MFE.  If you start it with ``make dev.up.lms`` that should give you
-everything you need as a companion to this frontend.
-
-Note that it is also possible to use `Tutor`_ to develop an MFE.  You can refer
-to the `relevant tutor-mfe documentation`_ to get started using it.
-
-.. _Devstack: https://github.com/openedx/devstack
+The Tutor_ platform is a prerequisite for developing an MFE.
+Utilize `relevant tutor-mfe documentation`_ to guide you through
+the process of MFE development within the Tutor environment.
 
 .. _Tutor: https://github.com/overhangio/tutor
 
 .. _relevant tutor-mfe documentation: https://github.com/overhangio/tutor-mfe#mfe-development
 
+
 Cloning and Startup
 ===================
 
-In the following steps, replace "[PLACEHOLDER]" with the name of the repo you
-created when copying this template above.
 
-1. Clone your new repo:
+1. Clone the repo:
 
-  ``git clone https://github.com/openedx/frontend-app-[PLACEHOLDER].git``
+  ``git clone https://github.com/openedx/frontend-app-catalog.git``
 
-2. Use node v18.x.
+2. Use node v20.x.
 
-   The current version of the micro-frontend build scripts support node 18.
-   Using other major versions of node *may* work, but this is unsupported.  For
-   convenience, this repository includes an .nvmrc file to help in setting the
-   correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
+  The current version of the micro-frontend build scripts support node 20.
+  Using other major versions of node *may* work, but this is unsupported.  For
+  convenience, this repository includes an .nvmrc file to help in setting the
+  correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
 
 3. Install npm dependencies:
 
-  ``cd frontend-app-[PLACEHOLDER] && npm install``
+  ``cd frontend-app-catalog && npm install``
```

Comment:
We should probably recommend `npm ci` here so people don't end up with lockfile changes the first time they try running this

Reply:
Agree, added